netci.groovy

```diff
@@ -433,7 +437,7 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
                     }
                 }
                 // For x86, only add per-commit jobs for Windows
-                else if (architecture == 'x86') {
```

Comment:
I think we are good not having pr1r2r legacy_backend runs per commit. If we really want r2r jobs for legacy_backend it is probably worth doing once a day or once a week. @RussKeldorph may have a different opinion.